src/pathways/common/services/lms-client.service.ts

```
@@ -485,60 +485,37 @@ export class LmsClientService {
-            return { courseId, ok: false, alreadyEnrolled: false };
-          } catch (err) {
-            failedCourseIds.push(courseId);
-            const msg = err instanceof Error ? err.message : 'Unknown error';
-            this.logger.warn(
-              `LMS enrollment error for user ${userId} course ${courseId}: ${msg}`
-            );
-            return { courseId, ok: false, alreadyEnrolled: false };
-          }
-        })
+    try {
+      this.logger.debug(`Enrolling user ${userId} to ${courseIds.length} courses in bulk`);
```

ESLint

[error] Replace ``Enrolling·user·${userId}·to·${courseIds.length}·courses·in·bulk`` with `⏎········`Enrolling·user·${userId}·to·${courseIds.length}·courses·in·bulk`⏎······`

(prettier/prettier)